src/networking.c

```diff
@@ -2801,7 +2853,8 @@ sds catClientInfoString(sds s, client *client) {
         client->lastcmd ? client->lastcmd->fullname : "NULL",
         client->user ? client->user->name : "(superuser)",
         (client->flags & CLIENT_TRACKING) ? (long long) client->client_tracking_redirection : -1,
-        client->resp);
+        client->resp,
+        (int) dictSize(client->meta));
```

Comment:
maybe we should list the whole meta string (with some escaping?), and not just it's length?
otherwise, we need some command to get the meta of other connections (not just the current one)